PWGCF/Flow/Tasks/flowEventPlane.cxx

```diff
@@ -131,6 +139,16 @@ struct FlowEventPlane {
     {"hYZNCVsCent", "hYZNCVsVx", "hYZNCVsVy", "hYZNCVsVz"}};
   std::map<CorrectionType, std::vector<std::vector<std::string>>> corrTypeHistNameMap = {{kFineCorr, vFineCorrHistNames}, {kCoarseCorr, vCoarseCorrHistNames}};
 
+  // Container for histograms
+  struct CorrHistContainer {
+    TH2F* hGainCalib;
+    std::array<std::array<THnSparseF*, 1>, 4> vCoarseCorrHist;
+    std::array<std::array<TProfile*, 4>, 4> vFineCorrHist;
+  } CorrHistContainer;
```

Comment:
Names are cheap in programming!
I would suggest `Correction` instead of `Corr` to distinguish from `Correlation`